### PR TITLE
Update How-tos for Pike 2

### PR DESCRIPTION
**Pull request by dnewh**
This updates the how-to docs to work with Pike 2.0. This includes
updating yaml files to include owners, updating endpoints, and using
updated commands for organizations, agents, and roles.

Signed-off-by: Davey Newhall <newhall@bitwise.io>

### CODE REVIEW

File: docs/0.2/creating_organizations.md [hunk 1]
```diff
@@ -231,6 +256,12 @@ in the example environment.
    $ curl http://localhost:8080/agent?service_id=01234-ABCDE::gsAA
    ```
 
+1. Request the list of roless.
```
Comment (dplumb94) [inline docs/0.2/creating_organizations.md:259]: typo

**Review by dplumb94 — APPROVED** [review]

**Review by dplumb94 — CHANGES_REQUESTED** [review]
Typo in one of the steps. Also it might be nice to use something else to differentiate the Pike ID vs the name. maybe the ID could be "myorg" and the name "MyOrganization" or something

**Comment by dplumb94** [comment]
We also need a How To procedure for Pike specifically since there is some more complex functionality now. I would recommend writing one based on the example from the RFC which hits on all of the features. Also would be good to include how to use the alternate ID index to fetch orgs by alternate ID.

File: docs/0.2/creating_organizations.md (hunk 1 above)
Comment (dnewh) [reply docs/0.2/creating_organizations.md]: fixed

**Review by RyanLassigBanks — DISMISSED** [review]

**Review by dplumb94 — APPROVED** [review]

**Review by RyanLassigBanks — APPROVED** [review]

**Pull request merged**